src/pages/patientView/genomicOverview/genomicOverviewHelper.ts

```diff
@@ -0,0 +1,301 @@
+/*
```

Comment:
we should make sure to run code formatting on this file as i think it's out of sync with our indentation convention